kbase-extension/static/kbase/js/widgets/appWidgets2/input/customInput.js

```diff
@@ -1,6 +1,6 @@
 define([
     'bluebird',
-    '../validators/text',
+    '../validation',
```

Comment:
There's a few of these. Break away from the inconsistently-used `validators/text` and go back to `validation`. This expects the `constraints` subobject itself as input, and does not return a Promise - there are a few cases where they need to get wrapped into a Promise later on.